tests/devices/qutrit_mixed/conftest.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Pytest configuration file for PennyLane qutrit mixed state test suite."""
+import numpy as np
+from scipy.stats import unitary_group
+import pytest
+
+SEED = 4774
+
+
+def get_random_mixed_state(num_qutrits):
+    dim = 3**num_qutrits
+
+    np.random.seed(seed=SEED)
+    basis = unitary_group.rvs(dim)
+    Schmidt_weights = np.random.dirichlet(np.ones(dim), size=1).astype(complex)[0]
```

Comment:
```suggestion
    rng = np.random.default(seed=4774)
    basis = unitary_group.rvs(dim, seed=584545)
    Schmidt_weights = rng.dirichlet(np.ones(dim), size=1).astype(complex)[0]
```


I (and numpy) generally recommend using a local random number generator whenever possible. We do sometimes run into issues where the tests get run in slightly different order, and then global state is in a slightly different position, and then suddenly our noisy test becomes too noisy and we get failures.

Fine to leave as is, especially as these tests aren't flaky, but thought I'd just make the recommendation anyway.

Reply:
Great, thank you I didn't know that, makes sense though!